services/121-service/src/main.ts

```diff
@@ -71,6 +71,15 @@ function generateModuleDependencyGraph(app: INestApplication): void {
 async function bootstrap(): Promise<void> {
   const app = await NestFactory.create(ApplicationModule);
 
+  if (process.env.NODE_ENV !== 'development' && !process.env.ENV_NAME) {
```

Comment:
There is already the const `DEBUG` that can be used here, instead of a special custom string-check.